scripts/bump_cloudbeat.sh

```diff
@@ -6,6 +6,7 @@ CURRENT_MINOR_VERSION=$(echo "$CURRENT_CLOUDBEAT_VERSION" | cut -d '.' -f1,2)
 export CURRENT_MINOR_VERSION
 
 # branches
+export BASE_BRANCH="${GIT_BASE_BRANCH:-main}"
```

Comment:
Since the bump version only works with the `workflow_dispatch` trigger, you can select any branch, including unconventional ones that may not be relevant for the bump process. I think we should add some checks on the `BASE_BRANCH` and restrict the script or workflow action `Bump Cloudbeat` to run only for specific patterns, such as `main`, `8.`, and `8.x`.

Reply:
That's a good idea, I've added a `validate_base_branch()` function to help with that.